hasura/metadata/actions.graphql

```diff
@@ -2,8 +2,6 @@ type Mutation {
   adminUpdateUser(payload: AdminUpdateUserInput!): UserResponse
 }
 
-# ignore; fake commit for triggering workflow
```

Comment:
@META-DREAMER this got merged from previous PR, so removing now. https://github.com/coordinape/coordinape/pull/664/commits/bf3eb2a9101e348c3c04d9bab243c04dcd72ca57